applications/callflow/doc/park.md

```diff
@@ -0,0 +1,84 @@
+## Call Parking
+
+### About Call Parking
+
+Parking will place a call in a numbered "slot" where it will remain until it is retrieved or the caller hangs up.
+
+### Schema for callflow action
+
+#### Mandatory fields
+**action** - action (see below)
+
+#### Optional fields
+
+* **{ACTION}**: default is `park`
```

Comment:
@jamesaimonetti its not an api.... and this is the format you gave me earlier...